FrontEnd/main/src/components/cookieacception/CookieMod.js

```diff
@@ -0,0 +1,54 @@
+import { Cookies } from 'react-cookie';
+import { useCookies } from 'react-cookie';
+import styles from './CookieMod.module.css';
+import cookiesimage from './cookieimg/homecookies.png';
+
+const CookieMod = ({ active, setActive }) => {
+  const [cookies, setCookie, removeCookie] = useCookies();
+  console.log(cookies.first);
+  const accessCookie = () => {
+    let d = new Date();
+    d.setTime(d.getTime() + 30 * 24 * 60 * 60 * 1000);
+    setCookie('first', true, { expires: d, sameSite: 'lax' });
+    setActive(false);
+  };
+
+  const denyCookie = () => {
+    let d = new Date();
+    d.setTime(d.getTime() + 30 * 24 * 60 * 60 * 1000);
+    setCookie('first', false, { expires: d, sameSite: 'lax' });
+    setActive(false);
+  };
+  return cookies.first ? (
+    ''
+  ) : (
+    <div
```

Comment:
Designers built designs using ant design. It think it makes sense using components like [modals](https://ant.design/components/modal) from there

Reply:
Thank you for feedback. But there was no such requirement in my task. My mentors and team knew about my design before the creation of this component

Reply:
I have discussed this issue with the mentor, and we decided to try ant design with the next task. Thank you for the advice.